packages/material-ui/src/Breadcrumbs/breadcrumbsClasses.d.ts

```diff
@@ -0,0 +1,12 @@
+export interface BreadcrumbsClasses {
+  root: string;
+  ol: string;
+  li: string;
+  separator: string;
```

Comment:
```suggestion
  ol: string;
  separator: string;
```

Shouldn't this look like this now?

Reply:
Although we aren’t reserving slots for `root` and `li`, we’re still giving them classes, so they should remain, I think?

Reply:
Got ya, correct, I confused them with the `root` from the other component 👍